boilerplate/app/components/AutoImage.tsx

```diff
@@ -10,6 +10,9 @@ export interface AutoImageProps extends ImageProps {
    * How tall should the image be?
    */
   maxHeight?: number
+  headers?: {
+    [key: string]: string
+  }
```

Comment:
Could you please document this to be consistent with the other props? Something mentioning that these are the headers for the request